brian2/devices/cpp_standalone/templates/network.cpp

```diff
@@ -1,3 +1,4 @@
+{# IS_OPENMP_COMPATIBLE #}
```

Comment:
Indeed, this (and a few more of the same) must have slipped in during the merge.